ruby-challenges.rb

```diff
@@ -9,8 +9,16 @@
 # Expected output: '42 is even'
 num3 = 221
 # Expected output: '221 is odd'
+def even_or_odd num
+    num % 2 == 0 ? "#{num} is even" : "#{num} is odd"
```

Comment:
Love the use of ternary operator here!  Take a look at the Ruby built in methods .even? and .odd?